docs/firststeps/loggingin.md

```diff
@@ -58,7 +58,7 @@ output enabled to get more information about what happens when you try to connec
 ssh -vvv -i <path-to-private-key> <username>@lumi.csc.fi
 ```
 
-If you are unable to connect and you contact the [user support team][helpdesk],
+If you are unable to connect, and you contact the [user support team][helpdesk],
```

Comment:
```suggestion
If you are unable to connect, contact the [user support team][helpdesk],
```

Reply:
If you look at the following line, this would become "connect, contact the user support team, we recommend"?